web/templates/invite_user_modal.hbs

```diff
@@ -40,16 +40,16 @@
                 <option {{#if this.default }}selected{{/if}} name="expires_in" value="{{this.value}}">{{this.description}}</option>
             {{/each}}
         </select>
-        <p id="expires_on"></p>
-        <div id="custom-invite-expiration-time" class="dependent-settings-block">
+        <p class="time-input-formatted-description"></p>
```

Comment:
This needs to be updated in the code that uses the `"expires_on"` id as changing only this will break the modal.